### DISCUSSION TITLE
The plug-in is not working after running as admin

### DISCUSSION THREAD

**Discussion by kimwooram502** [fredemmott/StreamDeck-Discord#57] [Q&A]
I set it up perfectly by referring to the YouTube video.
Until March 4th, the plug-in was working perfectly.
Plug-in does not work after running admin rights briefly due to discode overlay on March 5th.
The connection fails even if the debug information appears as a connection failure and the app is deleted from the DEVELOPER PORTAL site and reset.......
Re-enable Streamdeck and release the Discord Manager privilege will not resolve.
In this case, do I just delete the discode plug-in and install it again?
How can I delete the plug-in?

**Comment by Tetraplasm** [comment]
I'm in the same boat. I've tried uninstalling the plugin (you go to the stream deck app -> more actions and you can uninstall it from there) to no avail.

**Comment by fredemmott** [comment]
Assuming you're on windows, running as administrator is known to cause problems without known ways to fix them - however, currentlym, I'm not able to reproduce any persistent problems on latest windows 10 and discord, either runnign streamdeck or discord as admin.

That said, the plugin does not work if discord is *currently* running as admin; make sure you have fully quit discord, not just closed the window - open the system tray from the menu in the bottom right, select quit, then re-start discord *not* as admin.
